osgp/shared/osgp-ws-smartmetering/src/main/resources/schemas/sm-management.xsd

```diff
@@ -291,6 +291,7 @@
   <xsd:simpleType name="EventLogCategory">
     <xsd:restriction base="xsd:string">
       <xsd:enumeration value="STANDARD_EVENT_LOG"/>
+      <xsd:enumeration value="POWER_FAILURE_LOG"/>
```

Comment:
This adds a value to a schema based enumeration (`org.opensmartgridplatform.adapter.ws.schema.smartmetering.management`) which is mapped to or from a domain enumeration (`org.opensmartgridplatform.domain.core.valueobjects.smartmetering.EventLogCategory`).
Without changes to the domain enumeration I would expect errors can occur in the mapping as done in the EventConverter (`org.opensmartgridplatform.adapter.ws.smartmetering.application.mapping.EventConverter`).
Another related class is the DTO enumeration (org.opensmartgridplatform.dto.valueobjects.smartmetering.EventLogCategoryDto).
The DLMS protocol adapter has a command executor (`org.opensmartgridplatform.adapter.protocol.dlms.domain.commands.misc.FindEventsCommandExecutor`) where the eventlog category is mapped to an OBIS code for a particular event log.
If the new category reaches this command executor, changes are needed there as well to prevent exceptions.